MessageBroker.js

```diff
@@ -470,9 +470,22 @@ class MessageBroker {
 			if (msg.eventType == "custom/myVTT/audioPlayingSyncMe") {
 				self.handleAudioPlayingSync(msg);
 			}
-			if(msg.eventType == "character-sheet/character-update/fulfilled"){
-				if(window.DM)
-					self.handleCharacterUpdate(msg);
+			if(msg.eventType.includes('character-update')){
+				if(window.DM) {
+					
+					if(window.characterUpdatedRecently == undefined){
+						window.characterUpdatedRecently = [];
```

Comment:
ok - this is me not knowing how to do that tbh haha. I'll look into it unless its something you can help with real quick.

Reply:
ok I just removed the timeout stuff and left in handleCharacterUpdate - if it needs other changes to do what you want let me know

Reply:
Maybe I can be less picky about classes and mutation targets too then. Just if the HP Box, Ac box, Conditions box etc. I won't have to worry about multiple mutations triggering a message.

Reply:
Ok I feel like I'm doing something wrong here - I pushed the changes I thought would be required but it feels like it's working more like a timeout.

![what am I doing wrong here](https://user-images.githubusercontent.com/65363489/224317209-4a23928b-194e-4629-9727-3f8a0ea2583f.gif)

Reply:
> ,

well this makes complete sense - I should have realized that since it's a function lol. Thanks 

I also just noticed it doesn't seem like the mutation observer doesn't get loaded on a refresh when in game.